Modules/_sqlite/cache.c

```diff
@@ -52,6 +52,7 @@ void pysqlite_node_dealloc(pysqlite_Node* self)
     Py_DECREF(self->data);
 
     Py_TYPE(self)->tp_free((PyObject*)self);
+    Py_DECREF(pysqlite_NodeType);
```

Comment:
I would prefer to write something like:

```
    PyTypeObject *tp = Py_TYPE(self);
    tp->tp_free(self);
    Py_DECREF(tp);
```

To prepare the code base to port _sqlite3 to multi-phase init.

For comment for other dealloc functions.

Reply:
Yes, that's better.

Reply:
Fixed in 6be8abc2a8e6fabeffe29d16ef0fb78e154be61d